extensions/podman/packages/api/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@podman-desktop/podman-extension-api",
+  "version": "0.0.1",
+  "description": "API for Podman Desktop Podman extension",
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "types": "./src/podman-extension-api.d.ts",
+  "files": [
+    "src"
+  ],
+  "scripts": {
+    "prepare": "",
+    "clean": "rimraf lib *.tsbuildinfo",
+    "build": "",
+    "watch": "",
+    "publish:next": "yarn publish  --registry=https://registry.npmjs.org/ --no-git-tag-version --new-version 0.0.1-\"$(date +%s)\""
+  },
+  "dependencies": {
+    "@podman-desktop/api": "0.0.1"
```

Comment:
this dependency looks odd, because if I have a separate extension project, I will probably depends on a specific version of Podman Desktop

like depends on `"@podman-desktop/api": "1.12.0"` and there it'll bring another dependency to a published 0.0.1 which is not expected

Reply:
I guess we're missing in release and next-build an override of the version to match the version we're publising

Reply:
PTAL @dgolovin

Reply:
we'll miss the update on every release

we need to have the version being set when publishing

Reply:
Oh, of course. I was looking at the last line only (>.<). Thanks for pointing that out.

Reply:
While dependency to @podman-desktop/api should be definitely updated to ^1.12.0, I am still not sure under what version api package should be published. 
Does it make sense to increment it every build even there are no changes in the api package?
Should it be just the same as @podman-desktop/api?

Reply:
> Does it make sense to increment it every build even there are no changes in the api package?

we never know what will happen.

but it could also means that we would have inconsistent graph


like: myExtension depending on 'podman-api' ^1.10 and myExtension depending on 'podman-ext" : 1.10 it would also get a transitive dependency through podman-api to a different "podman-ext" version


so probably yes a safe path is to use the same version


> While dependency to @podman-desktop/api should be definitely updated to ^1.12.0, I am still not sure under what version api package should be published.
> Does it make sense to increment it every build even there are no changes in the api package?

I think it's ok as the lifecycle of the podman extension is the same than podman desktop for now.

so I'm not I would understand to use podman-api v1.3 and podman-desktop-api v1.12

Reply:
Dep to @podman-desktop/api is updated to `^1.12.0` in package.json.  I wonder if I can update it to latest podman-desktop release version during release running 

`yarn add @podman-desktop/api@${{ needs.tag.outputs.desktopVersion }}`

in release.yaml at release time